conf/app1.routes

```diff
@@ -0,0 +1,11 @@
+# microservice specific routes
```

Comment:
Small thing but could there be a more descriptive routes file name(s) now? something like `cafe.routes` and `secure-message.routes`?